presto-benchmark/src/main/java/com/facebook/presto/benchmark/CountAggregationBenchmark.java

```diff
@@ -43,7 +43,15 @@ public CountAggregationBenchmark(LocalQueryRunner localQueryRunner)
         OperatorFactory tableScanOperator = createTableScanOperator(0, new PlanNodeId("test"), "orders", "orderkey");
         InternalAggregationFunction countFunction = localQueryRunner.getMetadata().getFunctionRegistry().getAggregateFunctionImplementation(
                 new Signature("count", AGGREGATE, BIGINT.getTypeSignature()));
-        AggregationOperatorFactory aggregationOperator = new AggregationOperatorFactory(1, new PlanNodeId("test"), Step.SINGLE, ImmutableList.of(countFunction.bind(ImmutableList.of(0), Optional.empty())));
+        AggregationOperatorFactory aggregationOperator = new AggregationOperatorFactory(
```

Comment:
This is not a benchmark change. It's changed just because the constructor is changed.